api/tests/team/integration/infrastructure/repositories/certification-center-membership.repository.test.js

```diff
@@ -880,17 +881,20 @@ describe('Integration | Team | Infrastructure | Repository | Certification Cente
         const firstMembership = databaseBuilder.factory.buildCertificationCenterMembership({
           userId,
           certificationCenterId: firstCertificationCenterId,
-          createdAt: now,
+          createdAt: creationDate,
+          updatedAt: creationDate,
```

Comment:
```suggestion
          createdAt: creationDate,
```

Il est important de ne mettre dans les tests que ce qui est nécessaire. Ajouter dans ce test tout ces `updatedAt` n'a pas de rapport avec ce test. C'est pourquoi il faudrait éviter de les mettre.

Par contre comme nous en avons discuté il est tout à fait exact que d'avoir des valeurs logiques dans les objets manipulés par les tests est l'idéal. Aussi la solution me semble plutôt là aussi d'intervenir dans le database-builder plutôt que d'encombrer les tests. Voici une suggestion à mettre dans un commit supplémentaire : 

```javascript
const buildCertificationCenterMembership = function ({
  id = databaseBuffer.getNextId(),
  userId,
  updatedByUserId,
  certificationCenterId,
  createdAt = new Date(),
  updatedAt,
  disabledAt,
  isReferer = false,
  role = 'MEMBER',
} = {}) {
  userId = _.isUndefined(userId) ? buildUser().id : userId;
  certificationCenterId = _.isUndefined(certificationCenterId) ? buildCertificationCenter().id : certificationCenterId;
  updatedAt &&= createdAt
  if (disabledAt) {
    updatedAt = disabledAt;
  }

```

Reply:
Dans les autres tests j'ai supprimé le updatedAt mais dans celui-ci il me semble intéressant parce qu'il positionne updatedAt sur une valeur différente de celle qui est attendue après la désactivation.

Reply:
Avec la modification du database-builder que je propose ci-dessus le `updatedAt` serait positionné avec une autre valeur que celle attendue après la désactivation.